app/javascript/src/utils/editor.js

```diff
@@ -43,6 +43,19 @@ export function isAnyParentHidden(item) {
   return false
 }
 
+export function duplicateItem(item, newParent = null) {
+  const name = newParent ? item.name : item.name + " (Copy)"
```

Comment:
May be a little extra, but I think it's common for duplicated items to have a number increasing with each subsequent copy. Something like:
1. Check if there is an "{ item.name } (Copy)" under the same folder already
2. If so, use "{ item.name } (Copy 2)" instead
3. Check fi there is an "{ item.name } (Copy 2)" under the same folder already
4. If so, use "{ item.name } (Copy 3)" instead
5. Repeat as many times as necessary

Reply:
Good idea, I've changed it.

![duplicate-item-folder-count](https://user-images.githubusercontent.com/12848235/232313013-78de8353-1dae-497d-b619-1587da1650c3.gif)